### PR TITLE
Add com.github.naaando.lyrics

### PR DESCRIPTION
**Pull request by naaando**
Hi, my first app on flathub 😁️

### CODE REVIEW

File: com.github.naaando.lyrics.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{
+    "app-id": "com.github.naaando.lyrics",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.30",
```
Comment (LeandroStanger) [inline com.github.naaando.lyrics.json]: Update runtime to 3.32

**Comment by naaando** [comment]
Fixed

**Comment by nedrichards** [comment]
bot, build com.github.naaando.lyrics

File: com.github.naaando.lyrics.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.naaando.lyrics",
+    "build-options": {
+        "env": {
+            "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR": "/app/share/gir-1.0",
+            "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR": "/app/lib/girepository-1.0"
+        }
+    },
+    "finish-args": [
+        "--socket=x11",
```
Comment (nedrichards) [inline com.github.naaando.lyrics.json]: You need to add --share=ipc if you're using x11 per the docs http://docs.flatpak.org/en/latest/sandbox-permissions.html

Comment (naaando) [reply com.github.naaando.lyrics.json]: Confusingly it's working without it, I should be using this for display a GUI or this is meant for window managers, compositors, etc?

Comment (nedrichards) [reply com.github.naaando.lyrics.json]: It works, it just doesn't use X11 shared memory acceleration. You want the ipc option.

**Comment by naaando** [comment]
Solved, thanks.

**Comment by nedrichards** [comment]
bot, build com.github.naaando.lyrics

**Comment by nedrichards** [comment]
this seems to work well in local testing.

**Comment by naaando** [comment]
Ops! There's a space before the argument causing failed build, I'll fix
ASAP.

Em qua, 20 de mar de 2019 14:55, Flathub Bot <notifications@github.com>
escreveu:

**Comment by nedrichards** [comment]
bot, build com.github.naaando.lyrics

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.github.naaando.lyrics

**Pull request closed without merging**